streamlitDemo/README.md

```diff
@@ -1,14 +1,31 @@
-# NSMQ AI Workshop Demo
+# NSMQ AI Web App Environment Setup
 
-## Setup
-- The demo app can be run locally. Follow steps here to setup a python environment for streamlit https://docs.streamlit.io/library/get-started/installation
+### Prerequites
+1. Python 3 
+2. PIP 
 
-## Deploy
-- From the local repo directory `/nsmqai/streamlitDemo`, run command from the virtual environment terminal `python start_nsmqai.py`
-- WebApp will be deployed at `http://localhost:8501/`
+## Virtual Environment
+Select an environment manager of your choice to establish your virtual environment. Here's a selection of environment managers available for you to consider:
+- venv
+- virtualenv
+- conda
 
-![homepage](./images/homepage.png)
+### Is your virtual environment ready? If so, execute this command in your terminal which will set up the web application.
+```bash
+python start_nsmqai.py
+```
+
+Now, the web application will be running and accessible through the following URL: ```http://localhost:8501/```
+
+## API Setup 
+Here is the list of APIs that must be running to access the complete functionality of the web application. Note that each of these APIs is contained within their respective notebooks, and you need to have them running:
+- Speech-to-Text API
+- Text-to-Speech API
+- Question-Answering API
```

Comment:
@markowusu I think it will be worth adding the link to the various ML notebooks here-something to point them to the directory they have to go to locate the notebooks.